manual/cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/MergeTest.scala

```diff
@@ -231,7 +231,7 @@ return r""",
 merge (oliver)-[:DIRECTED]->(movie:Movie)<-[:ACTED_IN]-(reiner)
 return movie""",
       optionalResultExplanation = "In our example graph, 'Oliver Stone' and 'Rob Reiner' have never worked together. When we try to +MERGE+ a " +
-        "movie between them, Cypher will not use any of the existing movies already connected to either person. Instead, " +
+        "movie between them, Neo4j will not use any of the existing movies already connected to either person. Instead, " +
```

Comment:
I think it is better to keep it as "Cypher", rather than Neo4j, to emphasize that it is how the language operates...